druid-shell/src/mac/menu.rs

```diff
@@ -104,6 +104,12 @@ impl Menu {
     }
 }
 
+impl std::default::Default for Menu {
```

Comment:
I'm slightly confused by these. First, we don't need `std::default::`, it's already in the prelude. Second, is this saying we should prefer `default` to `new` in the simple case?

Reply:
Ah, this is an editor macro I have to make writing these impls easier, i didn't realize `Default` was in the prelude.

The text for this lint is basically "You have a zero-argument fn `new()`; people may expect to be able to use `default` here. In the one place where I strongly disagreed (around `WindowId`) I changed the name of the fn from `new` to `next`.

My feeling on these others is basically: the impls aren't _bad_, per-se, and I prefer adding them to adding a blanket `allow(...)` statement.